.github/workflows/ci.yml

```diff
@@ -71,6 +71,9 @@ jobs:
         pkg=oomplay-musl_${v}_amd64.deb
         wget https://github.com/oom-ai/oomplay/releases/download/v${v}/${pkg} &&
         sudo dpkg -i ${pkg}
+    - name: Install gotestsum
```

Comment:
Install it from binary ?

Reply:
Is it to standardize the installation in the ci?

Reply:
to make the ci faster